src/app/static/src/js/components/runReport/instances.vue

```diff
@@ -64,13 +69,14 @@
             selectInitialInstance: function () {
                 if (this.instances && this.showInstances) {
```

Comment:
Not new, but `this.instances` seems logically to be the same as `this.showInstances`